chia/rpc/rpc_server.py

```diff
@@ -19,6 +20,11 @@
 log = logging.getLogger(__name__)
 
 
+class RpcApiProtocol(Protocol):
+    def get_routes(self) -> Dict[str, Callable[[Any], Any]]:
+        pass
```

Comment:
I expect there will be more use cases for a more filled out version of this when we fill out more hints in more files.